apps/stable_diffusion/src/utils/utils.py

```diff
@@ -298,7 +298,8 @@ def set_init_device_flags():
         or args.batch_size != 1
         or ("vulkan" not in args.device and "cuda" not in args.device)
     ):
-        args.use_tuned = False
+       # args.use_tuned = False
```

Comment:
Instead of commenting this out (which would enable tuning for fp32 and all different sizes and possibly cause issues when we try to find tunings for sizes we don't have), you can change line 296-297 to include your cases more explicitly.
e.g.:
```
or (args.height == 512 and args.width not in [512, 768])
or (args.height == 768 and args.width not in [512, 768])
```